src/lib/battery/battery.cpp

```diff
@@ -153,7 +153,9 @@ battery_status_s Battery::getBatteryStatus()
 {
 	battery_status_s battery_status{};
 	battery_status.voltage_v = _voltage_v;
+	battery_status.voltage_filtered_v = _ocv_filter_v.getState();
 	battery_status.current_a = _current_a;
+	battery_status.current_filtered_a = _current_average_filter_a.getState();
```

Comment:
I would also just put the measurment `_current_a` here, the average filter has a very high time constant and is only used to estimate remaining flight time.